src/dayview/nowviewScript.ts

```diff
@@ -0,0 +1,53 @@
+export const getNowviewScript = () => `
```

Comment:
I'll move the nowview script to a separate branch for future study/use
feature/nowview-script

Reply:
Okay moved